### DISCUSSION TITLE
Checkbox 组件

### DISCUSSION THREAD

**Discussion by yangjinjun3** [jdf2e/nutui-react#7] [General]
功能、基本交互测试
 1、自定义尺寸时label字体大小没有随着变化
2、缺少半选效果
 ...
文档完整性验证
 ...
 ...
props 所有属性测试
 ...
 ...
props 异步修改组件是否动态更新
 ...
 ...
emits 所有事件测试
 ...
 ...
npm run build 子组件是否被构建
1、CheckBoxGroup 没有导出
 ...
其它
 1、import CheckBox from './checkbox';  checkbox是一个单词

**Comment by xiaoyatong** [comment]
1. 建议group 和 item 放在一个目录下即可。
2. 增加check的半选状态。
3. 命名：
![image](https://user-images.githubusercontent.com/84436086/139425593-312a7ca7-1e8e-4c51-b690-25a9d03bbece.png)
